### PR TITLE
Feature/sg 216 refactor seg aug dataloader

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: src/super_gradients/recipes/imagenet_repvgg.yaml [hunk 1]
```diff
@@ -4,20 +4,21 @@
 #  Reach => 72.05 Top1 accuracy.
 #
 #  Log and tensorboard at s3://deci-pretrained-models/repvggg-a0-imagenet-tensorboard/
-
+#
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_repvgg.yaml:7]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_vit_base.yaml [hunk 2]
```diff
@@ -6,11 +6,10 @@
 #  Log and tensorboard at s3://deci-pretrained-models/vit_base_imagenet1k/
 
 # Instructions:
-# running from the command line, set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/
-# Then:
-# for vit_base:
-#   python -m torch.distributed.launch --nproc_per_node=8 train_from_recipe_example/train_from_recipe.py --config-name=imagenet_vit_base
+#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_vit_base.yaml:9]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_vit_large.yaml [hunk 3]
```diff
@@ -6,10 +6,10 @@
 #  Log and tensorboard at s3://deci-pretrained-models/vit_large_cutmix_randaug_v2_lr=0.03/
 
 # Instructions:
-# running from the command line, set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/
-# Then:
-#   python -m torch.distributed.launch --nproc_per_node=8 train_from_recipe_example/train_from_recipe.py --config-name=imagenet_vit_large
+#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_vit_large.yaml:9]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml [hunk 4]
```diff
@@ -2,7 +2,13 @@ defaults:
   - default_train_params
 
 max_epochs: 250
-lr_updates: [100, 150, 200]
+
```
Comment (shaydeci) [inline src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml:5]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/training_hyperparams/default_train_params.yaml [hunk 5]
```diff
@@ -7,7 +7,11 @@ warmup_initial_lr: # Initial lr for linear_step. When none is given, initial_lr/
 step_lr_update_freq: # (float) update frequency in epoch units for computing lr_updates when lr_mode=`step`.
 cosine_final_lr_ratio: 0.01 # final learning rate ratio (only relevant when `lr_mode`='cosine')
 warmup_mode: linear_step # learning rate warmup scheme, currently only 'linear_step' is supported
-lr_updates: []
+
```
Comment (shaydeci) [inline src/super_gradients/recipes/training_hyperparams/default_train_params.yaml:10]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_resnet50_kd.yaml [hunk 6]
```diff
@@ -6,10 +6,11 @@
 #  Log and tensorboard at s3://deci-pretrained-models/KD_ResNet50_Beit_Base_ImageNet/average_model.pth
 
 # Instructions:
-# running from the command line, set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/:"YOUR_LOCAL_PATH"/super_gradients/src/
-# Then:
-#   python train_from_recipe_example/train_from_kd_recipe.py --config-name=imagenet_resnet50_kd
+#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_resnet50_kd.yaml:9]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_resnet50.yaml [hunk 7]
```diff
@@ -6,10 +6,10 @@
 #  Log and tensorboard at s3://deci-pretrained-models/ResNet50_ImageNet/average_model.pth
 
 # Instructions:
-# running from the command line, set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/
-# Then:
-#   python train_from_recipe_example/train_from_recipe.py --config-name=imagenet_resnet50
+#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_resnet50.yaml:9]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_regnetY.yaml [hunk 8]
```diff
@@ -6,21 +6,21 @@
 #  19 days for RegnetY600, 76.18
 #  20 days for RegnetY800, 77.07
 #  NOTE: Training should probably be lower as resources were shared among the above runs.
-
+#
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_regnetY.yaml:9]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_mobilenetv3_small.yaml [hunk 9]
```diff
@@ -0,0 +1,21 @@
+# MobileNetV3 Small Imagenet classification training:
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_mobilenetv3_small.yaml:1]: Don't think this file was supposed to be in this PR...

File: src/super_gradients/recipes/imagenet_mobilenetv3_base.yaml [hunk 10]
```diff
@@ -2,6 +2,7 @@
 
 defaults:
   - training_hyperparams: imagenet_mobilenetv3_train_params
+  - dataset_params: imagenet_dataset_params
```
Comment (shaydeci) [inline src/super_gradients/recipes/imagenet_mobilenetv3_base.yaml:5]: Don't think this file was supposed to be in this PR...

**Review by shaydeci — CHANGES_REQUESTED** [review]
I stopped after a few comments, there seems to be plenty of files that don't belong in this PR.

File: src/super_gradients/recipes/imagenet_repvgg.yaml (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_repvgg.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_vit_base.yaml (hunk 2 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_vit_base.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_vit_large.yaml (hunk 3 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_vit_large.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/training_hyperparams/default_train_params.yaml (hunk 5 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/training_hyperparams/default_train_params.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_resnet50.yaml (hunk 7 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_resnet50.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_mobilenetv3_small.yaml (hunk 9 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_mobilenetv3_small.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_mobilenetv3_base.yaml (hunk 10 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_mobilenetv3_base.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_resnet50_kd.yaml (hunk 6 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_resnet50_kd.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml (hunk 4 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/imagenet_regnetY.yaml (hunk 8 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/imagenet_regnetY.yaml]: This was part of previous PR that is now merged. Now it is removed fro this PR.

File: src/super_gradients/recipes/dataset_params/pascal_aug_segmentation.yaml [hunk 11]
```diff
@@ -0,0 +1,52 @@
+train_dataset_params:
```
Comment (shaydeci) [inline src/super_gradients/recipes/dataset_params/pascal_aug_segmentation.yaml:1]: Rename the file to pascal_aug_segmentation_dataset_params.yaml please (just as the rest...)

**Review by shaydeci — CHANGES_REQUESTED** [review]
Rename the file, solve conflicts and we can merge.

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**